packages/@office-iss/react-native-win32/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.win32.js

```diff
@@ -39,147 +65,286 @@ const _subscriptions = new Map();
  *
  * See https://reactnative.dev/docs/accessibilityinfo.html
  */
-
 const AccessibilityInfo = {
   /**
-   * iOS only
+   * Query whether bold text is currently enabled.
+   *
+   * Returns a promise which resolves to a boolean.
+   * The result is `true` when bold text is enabled and `false` otherwise.
+   *
+   * See https://reactnative.dev/docs/accessibilityinfo.html#isBoldTextEnabled
    */
-  isBoldTextEnabled: function(): Promise<boolean> {
-    return Promise.resolve(false);
+  isBoldTextEnabled(): Promise<boolean> {
+    if (Platform.OS === 'android' || Platform.OS === 'windows') {
```

Comment:
Note that `Platform.OS` returns `win32` instead of `windows` in the `react-native-win32` package.

Reply:
Same for below